Gemfile

```diff
@@ -1,5 +1,8 @@
 source 'https://rubygems.org'
 
+ruby RUBY_VERSION
+
+gem 'bundler'
```

Comment:
I can experiment with leaving it out - perhaps it is needed by the Rakefile, which gets read in our build.